src/components/Avatar.js

```diff
@@ -35,6 +44,14 @@ const defaultProps = {
 };
 
 class Avatar extends PureComponent {
+    avatarSource(imageStyle){
+        if (!this.props.network.isOffline) {
+            return(<Image source={{uri: this.props.source}} style={imageStyle}/>);
+        } else {
+           return(<Icon width={'100%'} height={'100%'} src={Profile} fill={colors.green}/>);
```

Comment:
I'm pretty certain this is not the way we should be doing this. 
We know that `AvatarWithIndicator` properly sets the offline image using the `isActiveProp`. Can you look into that and use the same strategy here?

Reply:
`AvatarWithIndicator` is used in 2 places in SettingsPage and in HomePage, and it takes `isActive` prop from parent.But `Avatar` component is used in a lot of different components and to use this strategy we have to add isActive prop in all of  `Avatar`'s parents components.Is this a good idea?

And we don't understand, at what way we can reproduce this issue. And is it necessary to do something with `Avatar` if it is displaying at correct way when device doesn't have connection.